core/providers/utils/fasthttpreplace_test.go

```
@@ -0,0 +1,214 @@
+)
+
+// parseFasthttpDirectives reports whether the module at modPath requires upstream
+// fasthttp, and whether it replaces it with the fork.
+//
+// go.mod is parsed with `go mod edit -json` rather than pattern-matched: require
+// and replace each have a single-line and a parenthesized-block form, and a
+// hand-rolled matcher that misses one silently skips the module - which in a
+// drift guard means reporting success on exactly the module it was added to
+// catch.
+func parseFasthttpDirectives(modPath string) (requires, replaced bool, err error) {
+	out, err := exec.Command("go", "mod", "edit", "-json", modPath).Output()
```

golangci-lint (2.13.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)